cmd/cagent/main.go

```diff
@@ -526,11 +604,18 @@ func getServiceFromFlags(ca *cagent.Cagent, configPath, userName string) (servic
 	return service.New(prg, svcConfig)
 }
 
-func setDefaultLogFormatter() {
+func setDefaultLogFormatter(ca *cagent.Cagent) {
 	tfmt := log.TextFormatter{FullTimestamp: true}
 	if runtime.GOOS == "windows" {
 		tfmt.DisableColors = true
 	}
 
 	log.SetFormatter(&tfmt)
+
```

Comment:
the name of the function now doesn't show what is happening inside

Reply:
@requilence It still configures default log formatter. Isn't ?

Reply:
from the other side, the function with name `setDefaultLogFormatter` now adds the file log hook which seems not quite expected

Reply:
> now adds the file log hook which seems not quite expected

agree

Reply:
done